cbor-file-magic.mkd

```diff
@@ -78,13 +81,19 @@ It is noted that in the media type registration, that a magic number is asked fo
 
 A challenge for the file(1) program is often that it can be confused by the encoding vs the content.
 For instance, an Android "apk" used to transfer and store an application may be identified as a ZIP file.
-Additionally, both OpenOffice and MSOffice files are ZIP files of XML files.
+Additionally, both OpenOffice and MSOffice files are ZIP files of XML files, and may also be identified as a ZIP file.
 
 As CBOR becomes a more and more common encoding for a wide variety of artifacts, identifying them as just "CBOR" is probably not sufficient.
 This document provides a way to encode a magic number into the beginning of a CBOR format file.
 Two possible methods of enveloping data are presented: a CBOR Protocol author will specify one.
 (A CBOR Protocol is a specification which uses CBOR as its encoding.)
 
+A third method is also proposed by which this CBOR format prepended tag is used to identify non-CBOR files.
+This third method has been placed in an appendix because it is not really about CBOR.
+
+This entire document is a Best Current Practice: it gives advice to designers of CBOR protocols on identifying their contents.
+This entire document is therefore informative, but the {{headers}} appendix is even more informative .
```

Comment:
The intention of this document is to define a standard for how to do this.  This doesn't get less normative by the existence of other standards in this space.  It's not a BCP because there is no C...

Reply:
This is now in #15